resources/hiveclients/java/hiveserver1-client/README.md

```diff
@@ -0,0 +1,26 @@
+#hiveserver1-client
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/96a5aee5c4e4100f587598fdf60f537307e184a0